autogl/solver/utils.py

```diff
@@ -1,23 +1,23 @@
 """
```

Comment:
First, thanks for fixing typos and bringing new output format. But remember, not everyone is happy when his/her designs or codes are modified without discussion or at least notification. Next time when you want to bring new features or change logic flow, you **should** first discuss with the one associated with the codes/classes.

There are also some conflicts with my design earlier, please find my comments below.